packages/workflow/package.json

```diff
@@ -35,6 +35,7 @@
     "./next": "./dist/next.js",
     "./nitro": "./dist/nitro.js",
     "./rollup-plugin": "./dist/rollup-plugin.js",
```

Comment:
we can change this to just `/rollup` in this PR if we want. might as well make the change now before waiting too long

I am worried we're polluting the `workflow` package with too many framework builders but we can break it apart in a future major release. changing `workflow/nitro` to `@workflow/nitro` is a pretty simple codemod